static/css/home.css

```
@@ -160,3 +160,33 @@ a.learn-more:hover {
     padding: 20px;
     width: auto;
 }
+
+.description-padding {
+    padding-top: 30px;
+    max-width: 500px;
+    width: 90%;
+    margin: 0 auto;
+}
+
+.portfolio-padding {
+    padding-bottom: 20px;
+    max-width: 500px;
+    width: 90%;
+    margin: 0 auto;
+}
+
+.youtube-responsive-container {
+    position:relative;
```

nit: space between CSS attribute and value:
```suggestion
    position: relative;
```
_[same with other values in this and the next selector]_